aips/aip-4.md

```diff
@@ -0,0 +1,149 @@
+---
+aip: 4
+title: Update Simple Map To Save Gas
+author: areshand
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/15
+Status: Review
```

Comment:
let's put this as draft stage and we will update to review once we get enough maintainer support

Reply:
updated